buildenv/Build_Instructions_V9.md

```diff
@@ -0,0 +1,375 @@
+make all
+```
+:warning: If you just type `make`, rather than `make all` your build will fail, because the default `make` target is `exploded-image`. If you want to specify `make` instead of `make all`, you must add `--default-make-target=images` when you run the configure script.
+
+Two Java builds are produced: a full developer kit (jdk) and a runtime environment (jre)
+- **build/linux-x86_64-normal-server-release/images/jdk**
+- **build/linux-x86_64-normal-server-release/images/jre**
+
+    :whale: If you built your binaries in a Docker container, copy the binaries to the containers **/root/hostdir** directory so that you can access them on your local system. You'll find them in the directory you set for `<host_directory>` when you started your Docker container. See [Setting up your build environment with Docker](#setting-up-your-build-environment-with-docker).
+
+    :pencil: On other architectures the **/jdk** and **/jre** directories are in **build/linux-ppc64le-normal-server-release/images** (Linux on 64-bit Power systems) and **build/linux-s390x-normal-server-release/images** (Linux on 64-bit z Systems)
+
+### 5. Test
+:penguin:
+For a simple test, try running the `java -version` command.
+Change to the /jre directory:
+```
+cd build/linux-x86_64-normal-server-release/images/jre
+```
+Run:
+```
+./bin/java -version
+```
+
+Here is some sample output:
+
+```
+openjdk version "9-internal"
+OpenJDK Runtime Environment (build 9-internal+0-adhoc.openj9-openjdk-jdk9)
+Eclipse OpenJ9 VM (build 2.9, JRE 9 Linux amd64-64 Compressed References 20171030_000000 (JIT enabled, AOT enabled)
+OpenJ9   - 731f323
+OMR      - 7c3d3d7
+OpenJDK  - 1983043 based on jdk-9+181)
+```
+:penguin: *Congratulations!* :tada:
+
+----------------------------------
+
+## AIX
+:blue_book:
+
+:construction:
+This section is still under construction. Further contributions expected.
+
+The following instructions guide you through the process of building an OpenJDK V9 binary that contains Eclipse OpenJ9 on AIX 7.2.
+
+### 1. Prepare your system
+:blue_book:
+You must install the following AIX Licensed Program Products (LPPs):
+
+- [Java8_64](https://adoptopenjdk.net/releases.html?variant=openjdk8#ppc64_aix)
+- [xlc/C++ 13.1.3](https://www.ibm.com/developerworks/downloads/r/xlcplusaix/)
+- x11.adt.ext
+
+A number of RPM packages are also required. The easiest method for installing these packages is to use `yum`, because `yum` takes care of any additional dependent packages for you.
+
+Download the following file: [yum_install_aix-ppc64.txt](aix/jdk9/yum_install_aix-ppc64.txt)
+
+This file contains a list of required RPM packages that you can install by specifying the following command:
+```
+yum shell yum_install_aix-ppc64.txt
+```
+
+It is important to take the list of package dependencies from this file because it is kept right up to date by our developers. However, if you want to understand the additional packages that `yum` installs, the list looks something like this:
```

Comment:
Please remove this section - this will be stale the first time the `yum_install_aix-ppc64.txt` file is updated.  Also, we don't list the additional packages pulled in when doing an `apt-get` so we don't need to do this here either.

Reply:
Done